test/function_test.py

```diff
@@ -993,3 +999,15 @@ def test_spawn_extended_feature_flag(client, servicer, monkeypatch, feature_flag
     else:
         expected_invocation_type = api_pb2.FUNCTION_CALL_INVOCATION_TYPE_ASYNC_LEGACY
     assert function_map.function_call_invocation_type == expected_invocation_type
+
+
+def test_spawn_grouped_function(client):
```

Comment:
Thanks for adding the test. definitely need to stop regressions happened while we iterate. 

There's already `i6pn_group_test.py` which is a good place to slot this in.